activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -102,19 +102,6 @@ class << t
           end
         end
 
-        def change_table(table_name, options = {})
-          if block_given?
-            super(table_name, options) do |t|
-              class << t
```

Comment:
`t` is a result of `update_table_definition`.
We need to prepend both `V5_0::TableDefinition` and `V4_2::TableDefinition` to `t` in `V4_2#change_table`.
If we remove the `change_table`, we will lost the V4_2 compatibility in `change_table`, I think.